schedule/sles4sap/migration/sles4sap_horizontal_migration.yaml

```diff
@@ -12,5 +12,5 @@ schedule:
   - console/system_prepare
   - sles4sap/migrate_sles_to_sles4sap
   - sles4sap/patterns
-  - sles4sap/sapconf
   - sles4sap/saptune
+  - sles4sap/sapconf
```

Comment:
Is the change also needed in the horizontal migration? Coming from SLES, the system should not have `saptune` enabled.

Reply:
I will remove it, but these tests keep failing on upload:
https://openqa.suse.de/tests/4725971#step/sapconf/55

Reply:
Could be another issue maybe, or a "real" timeout. I removed lot of `TIMEOUT_SCALE` with the YAML changes, to ensure to have this variable when it is really needed, and not in all tests as previously ;-)